actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -103,12 +115,13 @@ def initialize(params) # :nodoc:
   # You can fetch values of <tt>ActionController::Parameters</tt> using either
   # <tt>:key</tt> or <tt>"key"</tt>.
   #
-  #   params = ActionController::Parameters.new(key: 'value')
+  #   params = ActionController::Parameters.new(key: "value")
   #   params[:key]  # => "value"
   #   params["key"] # => "value"
   class Parameters
     cattr_accessor :permit_all_parameters, instance_accessor: false
     cattr_accessor :action_on_unpermitted_parameters, instance_accessor: false
+    cattr_accessor :raise_on_unpermitted_parameters, instance_accessor: false
```

Comment:
Do we need to define & deprecate this config value in 5.1, for people who upgrade from a 5.0 version that includes it?